parser/parser_test.go

```diff
@@ -84,20 +85,22 @@ func TestParseOutputFunc(t *testing.T) {
 }
 
 func TestParseCat(t *testing.T) {
+	dir := filepath.Join(os.Getenv("GOPATH"), "src/github.com/valyala/quicktemplate/parser")
```

Comment:
Use `build.Default.GOPATH` instead of `os.Getenv("GOPATH")`, since `GOPATH` environment variable may be missing starting from go1.8. See https://stackoverflow.com/a/44229943/274937 .